tests/unit/test_matrix.py

```
@@ -0,0 +1,54 @@
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from typing import Any
+
+import responses
+
+from viyapy import ViyaClient
+
+BASE = "https://viya.example.com"
+TOKEN = "test-token"
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "TOKEN"

(S105)